Move setup instructions to ur_client_library (#248)
UniversalRobots/Universal_Robots_Client_Library: doc/setup.rst

Setup a robot
=============

> 

Author: As this is not tied to the ROS drivers at the moment, I'd suggest removing it here. In the driver's documentation we have a [matching introduction](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/1245/files#diff-cbc6bcc6a61ac6d051c28c8a6c74a08652c01482442be78e6f4ef5c6bc256054R7)

```suggestion
To enable communication between the robot and the client library using the you must first prepare the robot and set up the robot correctly. This documentation will guide you through the necessary steps.
```

Setup a robot
=============

> To enable communication between the robot and the client library you must first prepare the robot
> and set up the robot correctly. This documentation will guide you through the necessary steps.
> 

Reviewer: ```suggestion
The robot has to be prepared, to enable communication between the robot and the client library.
This documentation will guide you through the necessary steps.

```
